modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -54,4 +54,14 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`spec.storage.managementState`
+
+|The Image Registry Operator sets the `spec.storage.managementState` parameter to `Managed` on new installations or upgrades of installer-provisioned infrastructure clusters on AWS or Azure.
+
+* `Managed`: Determines that the Image Registry Operator manages underlying storage. If the Image Registry Operator's `managementState` is set to `Removed`, then the storage is deleted.
+** If the `managementState` is set to `Managed` the Image Registry Operator attempts to apply some default configuration on the underlying storage unit. For example, if set to `Managed` the Operator tries to enable encryption on the S3 bucket before making it available to the registry. If you do not want the default settings to be applied on the storage you are providing make sure the `managementState` is set to `Unmanaged`.
```

Comment:
s/is set to `Managed` the/is set to `Managed`, the
s/if set to `Managed` the/if set to `Managed`, the
s/on the storage you are providing make/on the storage you are providing, make